charts/presidio/values.yaml

```diff
@@ -0,0 +1,52 @@
+registry: mcr.microsoft.com
+ 
+# Image pull secret
+# privateRegistry: acr-auth
+tag: latest
+
+# supported types are nginx (wip: traefik and istio)
+ingress:
+  enabled: true
+  class: nginx
+
+# For any of the services below, a service mesh like Linkerd or Istio
+# is needed to properly balance traffic to more than 1 replica.
+# https://github.com/microsoft/presidio/issues/304
+
+analyzer:
+  name: presidio-analyzer
+  replicas: 1
+  imagePullPolicy: Always
+  container:
+    resources:
+      requests:
+        memory: "1500Mi"
+        cpu: "1500m"
+      limits:
+        memory: "3000Mi"
+        cpu: "2000m"
+  service:
+    type: ClusterIP
+    externalPort: 80
+    internalPort: 8080
+    name: http
+
+anonymizer:
+  name: presidio-anonymizer
```

Comment:
missing resources, same for anonymizerimage.

Reply:
The only concern was the resources for the analyzer. So not sure resources limits are really required for them

Reply:
its a general best practice to have resource limits. vs code for instance has this as warning. its not required, but is better with. btw, specifically for presidio-image-redactor this could even be the same case as with analyzer, no?

Reply:
ok good point about the image redactor. I added the limits